src/schema/objects/suffixes.yaml

```
@@ -547,6 +548,15 @@ channels:
   display_name: Channels File
   description: |
     Channel information.
+colFA:
+  value: colFA
+  display_name: Colored Fractional Anisotropy image
+  description: |
+    Scanner-derived parametric map encoding both the anisotropy and principal direction of the diffusion tensor.
+    Image intensity is proportional to the Fractional Anisotropy (FA) measure;
+    image colour is determined by the principal eigenvector of the diffusion tensor,
```

Appease the spell checking gods (who are American):

```suggestion
    image color is determined by the principal eigenvector of the diffusion tensor,
```